web/viewer.html

```diff
@@ -27,6 +27,9 @@
 <!--#if GENERIC || CHROME-->
     <meta name="google" content="notranslate">
 <!--#endif-->
+<!--#if GENERIC || CHROME-->    
```

Comment:
This line should read `<!--#if GENERIC-->`, since this code is *not* needed in Chrome. Also, please remove the trailing whitespace at the end of this line.

Finally, you also need to squash the commits, please see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits.